treeshop.md

```diff
@@ -26,15 +26,20 @@ Clone this repository:
 
     git clone https://github.com/UCSC-Treehouse/pipelines.git
 
-Create a folders that match the [Treehouse storage layout](https://github.com/UCSC-Treehouse/pipelines/blob/master/fabfile.py#L12):
+Create needed directory  and navigate into the newly cloned repository:
+
+    mkdir .aws
+    cd pipelines
+
+Create folders that match the [Treehouse storage layout](https://github.com/UCSC-Treehouse/pipelines/blob/master/fabfile.py#L12):
 
     mkdir -p treeshop/primary/original/TEST treeshop/downstream
 
-Copy the TEST fastq samples into the storage hierarchy
-  
+Copy the TEST fastq samples into the storage hierarchy:
+
     cp samples/*.fastq.gz treeshop/primary/original/TEST/
 
-Spin up a single cluster machine:
+Spin up a single cluster machine (make sure you have created your SSH key):
```

Comment:
Provide instructions for:
a) checking if you already have an "~/.ssh/id_rsa" private+public keypair files that don't require a password
b) if you don't have a keypair, how to create it

Reply:
Another thing to incorporate at the end: 
- how to spin down machines when done
 - the info from Rob's April 12 email "Floating IPs" about how to clear the IP addresses after spinning down machines.